wikidict/convert.py

```diff
@@ -312,8 +312,50 @@ def process(self) -> None:
         self.summary(file)
 
 
-def get_formaters() -> List[Type[BaseFormat]]:
-    return [KoboFormat, DFFormat]
+class StarDictFormat(DictFileFormat):
+    """Save the data into a StarDict file."""
+
+    def _convert(self) -> None:
+        """Convert the DictFile to StarDict."""
+        from pyglossary import Glossary
+
+        Glossary.init()
+        glos = Glossary()
+
+        # Pretty print the source
+        source = wiktionary[self.locale].format(year=date.today().year)
+        glos.setInfo("description", source)
+        glos.setInfo("title", f"Wiktionary {self.locale.upper()}-{self.locale.upper()}")
+        glos.convert(
+            inputFilename=str(self.output_dir / f"dict-{self.locale}-{self.locale}.df"),
+            outputFilename=str(self.output_dir / "dict-data.ifo"),
+            writeOptions={"dictzip": False, "merge_syns": True},
```

Comment:
`merge_syns` was added mainly for "sdcv" client. Do you have reasons to enable it?

Reply:
Absolutely not, I'll remove it :)